requirements.in

```diff
@@ -18,7 +18,7 @@ cert_manager
 certsrv
 https://binaries.ddbuild.io/dd-source/python/cert_orchestration_adapter-0.1.2-py3-none-any.whl
 CloudFlare
-cryptography
+cryptography >= 41.0.4 # Required to avoid vulnerability in previous version (VULN-4474)
```

Comment:
I believe we'll need to re-generate `requirements.txt` as well. I believe the simplest way to do this would be to create a `virtualenv` then run `make up-reqs` from the root level directory of this repository, or use `pip-compile --upgrade` directly.

Reply:
was wondering about that - will go ahead and push a new iteration

Reply:
Pretty sure re-generating `requirements-dev.txt` should fix the most recent dependency conflict and fix the build so that this can then be merged.